packer/plugin_discover_test.go

```diff
@@ -347,62 +375,62 @@ func getFormattedInstalledPluginSuffix() string {
 
 var (
 	mockPlugins = map[string]pluginsdk.Set{
-		"bird": pluginsdk.Set{
+		"bird": {
 			Builders: map[string]packersdk.Builder{
 				"feather":   nil,
 				"guacamole": nil,
 			},
 		},
-		"chimney": pluginsdk.Set{
+		"chimney": {
 			PostProcessors: map[string]packersdk.PostProcessor{
 				"smoke": nil,
 			},
 		},
-		"data": pluginsdk.Set{
+		"data": {
 			Datasources: map[string]packersdk.Datasource{
 				"source": nil,
 			},
 		},
 	}
 	mockInstalledPlugins = map[string]pluginsdk.Set{
-		fmt.Sprintf("bird_%s", getFormattedInstalledPluginSuffix()): pluginsdk.Set{
+		fmt.Sprintf("bird_%s", getFormattedInstalledPluginSuffix()): {
 			Builders: map[string]packersdk.Builder{
 				"feather":   nil,
 				"guacamole": nil,
 			},
 		},
-		fmt.Sprintf("chimney_%s", getFormattedInstalledPluginSuffix()): pluginsdk.Set{
+		fmt.Sprintf("chimney_%s", getFormattedInstalledPluginSuffix()): {
 			PostProcessors: map[string]packersdk.PostProcessor{
 				"smoke": nil,
 			},
 		},
-		fmt.Sprintf("data_%s", getFormattedInstalledPluginSuffix()): pluginsdk.Set{
+		fmt.Sprintf("data_%s", getFormattedInstalledPluginSuffix()): {
 			Datasources: map[string]packersdk.Datasource{
 				"source": nil,
 			},
 		},
 	}
 
 	invalidInstalledPluginsMock = map[string]pluginsdk.Set{
-		"bird_v0.1.1_x5.0_wrong_architecture": pluginsdk.Set{
+		"bird_v0.1.1_x5.0_wrong_architecture": {
 			Builders: map[string]packersdk.Builder{
 				"feather":   nil,
 				"guacamole": nil,
 			},
 		},
-		"chimney_cool_ranch": pluginsdk.Set{
+		"chimney_cool_ranch": {
 			PostProcessors: map[string]packersdk.PostProcessor{
 				"smoke": nil,
 			},
 		},
-		"data": pluginsdk.Set{
+		"data": {
 			Datasources: map[string]packersdk.Datasource{
 				"source": nil,
 			},
 		},
 	}
 	defaultNameMock = map[string]pluginsdk.Set{
-		"foo": pluginsdk.Set{
```

Comment:
linter reported redundant type declaration, as the map already knows the expected type.